templates/AskdarcelApiService.json

```diff
@@ -90,7 +90,7 @@
 	  "Cpu": "1",
 	  "Essential": "true",
 	  "Image": { "Ref": "ContainerImage" },
-	  "Memory": "300",
+	  "Memory": "1000",
```

Comment:
Yea, makes sense.  I'll put a backlog item to review this, but for now I want to update the template with something I know works.